addons/project/static/src/views/components/project_task_template_dropdown.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="project.TemplateDropdown">
+        <button t-if="!taskTemplates.length" type="button" t-att-class="props.newButtonClasses" t-att-data-hotkey="props.hotkey" t-on-click.stop="props.onCreate" data-bounce-button="">New</button>
+        <t t-else="">
+            <button type="button" t-attf-class="{{ props.newButtonClasses }} dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" t-att-data-hotkey="props.hotkey" data-bounce-button="">
+                New
+            </button>
+            <ul class="dropdown-menu o-project-form-dropdown-menu">
+                <button type="button" class="btn btn-link dropdown-item o-dropdown-item" t-on-click="props.onCreate">
+                    New Task
+                </button>
+                <div class="dropdown-divider"/>
+                <div class="dropdown-header" style="padding-left: 20px;">Task Templates</div>
```

Comment:
Cannot we use the existing classes? Such as `ps-4`?
```suggestion
                <div class="dropdown-header" style="padding-left: 20px;">Task Templates</div>
```

Reply:
Alexandra wanted the same style as for the option menus, and that's how it's done there, so I thought that if we want something identical, I should probably do the same